scenarios/12_Leaving_Raedwood.cfg

```diff
@@ -262,6 +280,43 @@ Move Ruvio to signpost to ask him for advice."}
         {INTERROGATION12}
     [/event]
 
+    [event]
+        name=last breath
+        [filter]
+            id=Quivre O Raedbor
+        [/filter]
+        [filter_second]
+            id=Gawen Hagarthen
+        [/filter_second]
+        {MSGW_Ruvio _"Gawen! I thought I told you not to waste your time with that one!"}
+        {MSGW_Gawen _"I know, I was just curious to see if he had anything to say."}
```

Comment:
Also doesn't sound like Gawen to me. He speaks a bit more plainly.

Reply:
I don't really know how to make that any plainer though? Those are all pretty plain words...

Reply:
> I know, I was only curious.

Reply:
I used that in 6f04548 too.